modules/stanford_image_styles_preview/src/Form/PreviewForm.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Drupal\stanford_image_styles_preview\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Class PreviewForm.
+ */
+class PreviewForm extends FormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'preview_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form = [];
+
+    $form['image'] = [
+      '#type' => 'managed_file',
```

Comment:
My concern is more with orphaned files floating around the file system/table. If you can confirm that the image does get garbage collected I am good with this.